plugins/TagFix_MultipleTag.py

```diff
@@ -74,17 +74,17 @@ def init(self, logger):
         self.errors[1050] = self.def_class(item = 1050, level = 1, tags = ['highway', 'roundabout', 'fix:chair'],
             title = T_('Reverse roundabout'),
             detail = T_(
-'''The circulation of the roundabout is draw clockwise, but in countries
-where they drive on the right the sense of roundabouts is
-counterclockwise, and vice versa for the other countries.'''),
+'''The circulation of the roundabout is drawn clockwise, but in countries
+where they drive on the right, the circulation of roundabouts is
+counter clockwise, and vice versa for other countries.'''),
```

Comment:
https://en.wikipedia.org/wiki/Clockwise
counterclockwise seems to be 1 word

Reply:
...and https://github.com/osm-fr/osmose-backend/pull/1704/commits/5fd231b5553628ae7712ce0eabe9b5cf3d9586a3